extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -281,13 +331,21 @@ export class AmpIframe extends AMP.BaseElement {
    * @private
    */
   activateIframe_() {
-    this.getVsync().mutate(() => {
-      if (this.placeholder_) {
-        this.iframe_.style.zIndex = '';
-        removeElement(this.placeholder_);
-        this.placeholder_ = null;
-      }
-    });
+    if (this.placeholder_) {
+      this.getVsync().mutate(() => {
+        if (this.iframe_) {
+          this.iframe_.style.zIndex = 0;
```

Comment:
Ok. Let's stick with `0` then. I think it's actually better to have `''` in the long term because it'd fallback to `<style>` declarations. But since we control that part of CSS more or less completely, we should be ok. That does mean that something was wrong with the previous test.